SyncMyMod/autoinstall.sh

```diff
@@ -25,102 +25,92 @@
-APP_FOLDER="quake2"		# NO SPACE OR SPECIAL CHARS ALLOWED!!
-APP_FILE="Quake2.qml"		# NO SPACE OR SPECIAL CHARS ALLOWED, FIRST LETTER MUST BE IN UPPERCASE!!
-APP_HIDETITLE="true"		# ONLY TRUE OR FALSE ALLOWED!!
+APP_AUTHOR="auron89"        # FOLDER NAME USED TO GROUP APPS OF THE SAME DEVELOPER. NO SPACE OR SPECIAL CHARS ALLOWED!!
+APP_NAME="Quake 2"          # VISIBLE DURING INSTALLATION PROCESS AND FROM APPS LOADER. ALL CHARS ALLOWED BUT SOME SPECIAL CHARS MAY CAUSE ISSUES
+APP_FOLDER="quake2"         # APP FOLDER. NO SPACE OR SPECIAL CHARS ALLOWED!!
+APP_FILE="Quake2.qml"       # MAIN APP FILE. NO SPACE OR SPECIAL CHARS ALLOWED, FIRST LETTER MUST BE IN UPPERCASE!!
+APP_HIDETITLE="true"        # USED TO DEFINE IF TITLEBAR MUST REMAIN VISIBLE OR NOT. ONLY TRUE OR FALSE ALLOWED!!
+
+AUTHOR="BigUncleMax"        # DEVELOPER NAME VISIBLE DURING THE INSTALLATION PROCESS. ALL CHARS ALLOWED BUT SOME SPECIAL CHARS MAY CAUSE ISSUES
 
 ###############################################################################
-# Environment Variables                                                       #
+# Environment variables                                                       #
 ###############################################################################
 
-# Mod Name
-FANCYNAME="Quake 2"
-
-# Developer Name
-AUTHOR="BigUncleMax"
-
-# Required Mod
-DEPENDENCY="CUSTOM_APPS_LOADER"
-
-# FMods Tools 2.5 entry string
-MODTOOLS="FMODS_TOOLS_2.5"
+PATH=/fs/rwdata/dev:$PATH
 
 APIM_APPS_PATH=/fs/mp/fordhmi/qml/hmicustomapps/apps
 APIM_JSON_PATH=/fs/mp/fordhmi/qml/hmicustomapps/apps.json
 LOCAL_APP_PATH=/fs/usb0/SyncMyMod/app/
 
+DISPLAY=/fs/tmpfs/status
+POPUP=/tmp/popup.txt
+
+# FMods Tools entries
+MODTOOLS="FMODS_TOOLS"
+MIN_MODTOOLS_VERSION="2.8"
+
+DEPENDENCY="CUSTOM_APPS_LOADER"
+
 FILES_DIR="/fs/usb0/SyncMyMod/files"
 BIN_DIR="${FILES_DIR}/bin"
 SHELL_DIR="${FILES_DIR}/shell"
 OTHER_DIR="${FILES_DIR}/other"
 
-DISPLAY=/fs/tmpfs/status
-
 KEEP_SAVED_GAMES=1
 MOD_DATA_DIR="${OTHER_DIR}/quake2"
 FMODS_DATA_DIR=/fs/images/fmods_apps_data
 INSTALLATION_DIR=${FMODS_DATA_DIR}/Quake2/
 BACKUP_DIR=${FMODS_DATA_DIR}/Quake2_bak
 
-DISPLAY=/fs/tmpfs/status
-POPUP=/tmp/popup.txt
+VERSION=$(cat ${OTHER_DIR}/quake2/version.txt)
+PREV_VERSION=$(cat ${FMODS_DATA_DIR}/Quake2/version.txt)
 
 LOG_DIR="/fs/usb0"
 LOG_FILE=/tmp/install_status.txt
 HWINFO_FILE=/tmp/hardware_info.txt
 
-# Mod version
-VERSION=$(cat ${OTHER_DIR}/quake2/version.txt)
-PREV_VERSION=$(cat ${FMODS_DATA_DIR}/Quake2/version.txt)
-
 ###############################################################################
 # Functions                                                                   #
 ###############################################################################
-function dump_logs {
-	cp ${LOG_FILE} ${LOG_DIR}/install_status.txt
+dump_logs() {
+	cp "${LOG_FILE}" "${LOG_DIR}"/install_status.txt
 	sync
 }
 
-function output {
-	echo "${1}" > $DISPLAY
-	echo "${1}" >> $LOG_FILE
```

Comment:
Is it okay that we drop the log output here?

Reply:
i recreated the functions with ()